.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Ask a question (Matrix Channel)
+    url: https://github.com/esp-rs/esp-hal/discussions/new
+    about: Ask questions in our Matrix channel.
+  - name: Ask a question (GitHub Discussions)
```

Comment:
Do we want to mention discussions at all? Matrix is a better place for questions imo

Reply:
> Do we want to mention discussions at all?

I guess it depends on what we want to do with discussions. Just removed the GH discussion link, I also though about adding a "recommended" to the Matrix option, but for now I removed the GH discussion.

Reply:
> Another thing is discoverability of answered questions

True, but I guess I never search discussions on other projects :see_no_evil:, I don't mind discussions, I was just posing the question whether we really want them. I'll let others chime in and we can make a consensus.

Reply:
I think they're useful. People are opening new discussions and commenting on existing ones regularly, and these will just get opened as issues otherwise.

Reply:
People already open discussions as issues:
 - https://github.com/esp-rs/esp-hal/issues/2108
 - https://github.com/esp-rs/esp-hal/issues/2088
 - https://github.com/esp-rs/esp-hal/issues/1432

Currently, people also open issues as discussions:
 - https://github.com/esp-rs/esp-hal/discussions/2106

I'm pretty sure there are more examples, I'm too lazy to check. I'd rather comb through one list, where we can close stuff, instead of two. If we commit to sort through these misplaced entries, discussions will make more sense, but is that important enough to allocate time for?

Reply:
@playfulFence @JurajSadel any inputs on this topic?

Reply:
I kind of agree with everyone here - Having discussions might not be necessary since we have Matrix _but_ it's definitely easier to find/reference discussion than searching in Matrix conversation. 
Regarding having issues opened as discussions and vice versa they can be converted between each other and issue guard can take care of it when necessary? Just an idea

Reply:
Me personally, I'd leave discussions”... 
It seems to me to be the most obvious way to start, well, a discussion 😄. Also, as Jesse mentioned, people do use it. Personally, if I were a user, I'd rather use "centralized" GH than Matrix. But that's just me and as practice shows, Matrix is at least same popular way of communication.

Reply:
> Do we want to mention discussions at all?

Since most of us tend to agree on keeping GH discussion open, I think I should also add a link to it to avoid issues being used as discussions. We can mark the Matrix option as recommended or something like that.

Reply:
Sounds good, let's actively convert issues to discussions and vice versa where appropriate to try and alleviate the points made by @bugadani.